jade-signer-rs/src/rpc/serves.rs

```diff
@@ -321,12 +321,12 @@ pub fn sign_transaction(
 }
 
 pub fn sign(
-    params: Either<(SignData,), (SignData, CommonAdditional)>,
+    params: SignParams<(SignData, String), (SignData, String, CommonAdditional)>,
     storage: &Arc<Mutex<StorageController>>,
     wallet_manager: &Arc<Mutex<RefCell<WManager>>>,
 ) -> Result<Params, Error> {
     let storage_ctrl = storage.lock().unwrap();
-    let (input, additional) = params.into_full();
+    let (input, passphrase, additional) = params.into_full();
```

Comment:
`sign` should take 4 params.

`dataToSign`, `address`, `password`, `additional`

heres the web3 personal one

![image](https://user-images.githubusercontent.com/364566/58039337-572b4b80-7ae7-11e9-872a-fadecbd84d10.png)